pkg/clusterlink/controllers/node/node_controller.go

```diff
@@ -171,10 +171,10 @@ func (r *Reconciler) CleanOrphan() error {
 	}
 	var errs []error
 	cnt := 0
-	for _, cn := range clusterNodes.Items {
-		clusterNode := cn
-		if _, ok := k8sNodeNameSet[cn.Spec.NodeName]; !ok {
-			if err := r.Delete(context.Background(), &clusterNode); err != nil {
+	for i := range clusterNodes.Items {
+		clusterNode := clusterNodes.Items[i]
+		if _, ok := k8sNodeNameSet[clusterNode.Spec.NodeName]; !ok {
+			if err := r.ClusterLinkClient.KosmosV1alpha1().ClusterNodes().Delete(context.Background(), clusterNode.Name, metav1.DeleteOptions{}); err != nil {
 				errs = append(errs, err)
```

Comment:
Why  not use  `r.Delete` ?

Reply:
nodecontroller will watch member cluster node change and update cluster node in host cluster, so the r.Delete is connect to member cluster, and clusterlinkclient is connect to host cluster

Reply:
/lgtm